Fix typos and clarify language in atip-features.adoc (#1210)
suse-edge/suse-edge.github.io: asciidoc/product/atip-features.adoc

** `rcupdate.rcu_task_stall_timeout=0` disables the RCU task stall detector, preventing potential warnings or system halts from long-running RCU tasks.

> ** `rcutree.kthread_prio=99` sets the priority of the RCU callback kernel thread to the highest possible (99), ensuring it gets scheduled and handles RCU callbacks promptly, when needed.

Reviewer: If I may suggest here, 

```suggestion
** `rcutree.kthread_prio=99` sets the priority of the RCU callback kernel thread to the highest possible value (99), ensuring it is scheduled immediately to handle callbacks promptly when needed.
```

> * Add `ignition.platform.id=openstack` for Metal3 and Cluster API to successfully provision/deprovision the cluster. This is used by Metal3 Python agent, which originated from OpenStack Ironic.

Reviewer: Should it be Metal^3^? 
```suggestion
* Add `ignition.platform.id=openstack` for Metal^3^ and Cluster API to successfully provision/deprovision the cluster. This is used by Metal^3^ Python agent, which originated from OpenStack Ironic.
```

| nohz
| on

> | When enabled, kernel's periodic timer interrupt (the 'tick') will stop on any CPU core that is idle. This primarily benefits the housekeeping CPUs (`0,31,32,63`). This conserves power and reduces unnecessary wake-ups on those general-purpose cores.

Reviewer: ```suggestion
| When enabled, the kernel's periodic timer interrupt (the 'tick') stops on any CPU core that becomes idle. This primarily benefits the housekeeping CPUs (`0,31,32,63`), conserving power and reducing unnecessary wake-ups on those general-purpose cores.
```

----

==== Bond CNI with SR-IOV
Using the Bond CNI with SR-IOV is fairly straightforward. For more details on how to set up SR-IOV, see <<sriov>>. As described there, you have to create `SriovNetworkNodePolicies` that defines `resourceNames`, as well as number of virtual functions and such. The `resourceNames` are being used by the `SriovNetwork` which is used as interfaces in the pod definition. The bond definition is exactly the same as for the MACVLAN and host device cases.

[NOTE]
====

> Bond CNI with SR-IOV is only applicable to SRIOV Virtual Functions (VF) using the kernel driver. Userspace driver VFs - such as those used in DPDK workloads - cannnot be bonded with the Bond CNI.

Reviewer 2: I think we have one too many "n" characters here

Author: fixed!
